### DISCUSSION TITLE
pause experiment when exit fullscreen

### DISCUSSION THREAD

**Discussion by juliameehan** [jspsych/jsPsych#1627] [Q&A] [answered]
Hi! I am trying to write code for an experiment and want it so that when someone exits fullscreen the experiment pauses and the person has to reenter fullscreen for it to continue. Any ideas on how to do this? I currently can get the experiment to pause when exit fullscreen by using below: 

     on_interaction_data_update: function(data) {
        if (data.event == 'fullscreenexit'){
          jsPsych.pauseExperiment();
        }

Thanks so much!!

**Comment by becky-gilbert** [comment]
Hi @juliameehan! I just posted an [answer to this question](https://github.com/jspsych/jsPsych/discussions/1364#discussioncomment-476605) on a different discussion thread. Could you have a look at that and see if it works for you?

**Reply by juliameehan** [reply]
hi @becky-gilbert ! so I tried this code and no message is popping up to go back into fullscreen and resume the experiment (just remains paused). I am happy to add my code if that helps! It could be due to the issue I am having also with tracking fullscreen enter and exit. Thanks so much for your help!

**Comment by becky-gilbert** [comment, marked as answer]
@juliameehan it looks like you already found this, but FYI for anyone else trying to do the same thing: 
The answer I linked to in my other comment doesn't work. Please see [this comment](https://github.com/jspsych/jsPsych/discussions/1364#discussioncomment-490858) for a solution to this question.
